sdk/go/aws/directconnect/gatewayAssociation.go

```diff
@@ -143,8 +143,6 @@ import (
 // 	})
 // }
 // ```
-//
-// A full example of how to create a VPN Gateway in one AWS account, create a Direct Connect Gateway in a second AWS account, and associate the VPN Gateway with the Direct Connect Gateway via the `directconnect.GatewayAssociationProposal` and `directconnect.GatewayAssociation` resources can be found in [the `./examples/dx-gateway-cross-account-vgw-association` directory within the Github Repository](https://github.com/providers/provider-aws/tree/master/examples/dx-gateway-cross-account-vgw-association).
```

Comment:
We are losing this comment (in all languages).

Reply:
This is correct - it points directly to the terraform providers repo which we don't want